src/app/(nextjs_migration)/layout.tsx

```diff
@@ -47,7 +47,7 @@ export default async function MigrationLayout({
       {falseDoorFlag &&
         waitlistLink &&
         countryCode.toLowerCase() === "us" &&
-        locale === "en" && <HandleFalseDoorTest link={waitlistLink} />}
+        locale === "en-US" && <HandleFalseDoorTest link={waitlistLink} />}
```

Comment:
This covers the locales we currently support (`en` and `en-CA`), and the ones we might support in the future:

```suggestion
        ["en", "en-GB", "en-US", "en-CA"].includes(locale) && <HandleFalseDoorTest link={waitlistLink} />}
```

Reply:
Oh heh, [that was actually the old situation](https://github.com/mozilla/blurts-server/pull/3327/files). So to clarify, the old situation did [already look at the Accept-Language headers](https://github.com/mozilla/blurts-server/blob/d4ff67a2e8ea9ae63815f95676b566b24b8b6209/src/app/functions/server/l10n.ts#L51-L52), so I'm not quite sure why that was changed?